### PR TITLE
Update assets for new accounts

### PR DESCRIPTION
**Pull request by cartersocha**
Wasn't sure where these new accounts fit so created a new category

### CODE REVIEW

**Comment by cartersocha** [comment]
CC @arminru

File: assets.md [hunk 1]
```diff
@@ -10,6 +10,32 @@ This file is intended to list all the assets controlled by OpenTelemetry.
...
+- Admin: Juraci Krohling @jpkrohling
+
+### Grafana organization
+
+Link: https://grafana.com/orgs/otelsigsecurity
+
+- Used to provide an overview of open security incidents in the org and
+  historical data on how quickly we respond to incidents
+- Users: SIG-Security Maintainers, Technical and Governance Committees
+- Admins: Juraci Krohling @jpkrohling & Armin Ruech @arminru
```
Comment (trask) [inline assets.md:37]:
```suggestion
- Admins: Juraci Paixão Kröhling @jpkrohling & Armin Ruech @arminru
```

**Review by trask — APPROVED** [review]
thanks for documenting these!

**Comment by trask** [comment]
https://grafana.com/orgs/otelsigsecurity is failing the markdown-link-check, you can suppress in https://github.com/open-telemetry/community/blob/main/.github/scripts/markdown-link-check-config.json

File: assets.md [hunk 2]
```diff
@@ -10,6 +10,32 @@ This file is intended to list all the assets controlled by OpenTelemetry.
...
   - Admins: [@open-telemetry/governance-committee](https://github.com/orgs/open-telemetry/teams/governance-committee)
 
+## Community Resource Accounts
+
+### AWS account
+
+Link: http://cncf-aws-opentelemetry.signin.aws.amazon.com/
+
+- Community account to published Lambda layers
+- Admin: Alex Boten @codeboten & Anthony Mirabella @aneurysm9
```
Comment (arminru) [inline assets.md:20]: @open-telemetry/lambda-extension-maintainers could you request access to these as well?

File: assets.md [hunk 3]
```diff
@@ -10,6 +10,32 @@ This file is intended to list all the assets controlled by OpenTelemetry.
...
+
+### Equinix bare metal
+
+Link: https://console.equinix.com/projects/6f1c9af6-0470-42da-8f22-59d0df245f6b
+
+- Reserved bare metal machines for the community to run performance tests and
+  other assets on
+- Admin: Juraci Paixão Kröhling @jpkrohling
```
Comment (arminru) [inline assets.md:28]:
```suggestion
- Admin: Juraci Paixão Kröhling @jpkrohling and OTel TC via the mailing list address
```

File: assets.md [hunk 4]
```diff
@@ -10,6 +10,32 @@ This file is intended to list all the assets controlled by OpenTelemetry.
...
+
+### Grafana organization
```
Comment (arminru) [inline assets.md:30]:
```suggestion
### Grafana organization for SIG Security
```

**Review by arminru — APPROVED** [review]
Thank you!

**Comment by trask** [comment]
😂 

> [✖] https://status.linuxfoundation.org/ → Status: 0

File: assets.md (hunk 2 above)
Comment (tylerbenson) [reply assets.md:20]: How do we do so?

**Pull request merged**